modules/apps/frontend-js/frontend-js-spa-web/src/main/java/com/liferay/frontend/js/spa/web/internal/configuration/SPAConfiguration.java

```diff
@@ -42,6 +42,12 @@
 	)
 	public String[] navigationExceptionSelectors();
 
+	@Meta.AD(
+		description = "custom-excluded-paths-description",
+		name = "custom-excluded-paths-name", required = false
```

Comment:
First time seeing `@Meta.AD()` but shouldn't the `required = false` be on it's own line?

Reply:
You don't argue about SF, it's either green or it isn't ;)

Reply:
why?

Anyway, it is controlled by source format tools.

Reply:
Oh I see, interesting syntax 👍